src/functions/utils.php

```diff
@@ -1330,3 +1330,44 @@ function tribe_get_next_cached_increment( $key, $expiration_trigger = '', $defau
 
 	return $value;
 }
+
+if ( ! function_exists( 'tribe_copy_to_clipboard_button' ) ) {
+	/**
+	 * Output a button to copy the content of an element to the clipboard.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $content_to_copy The content to copy to the clipboard.
+	 * @param bool   $output_button   Whether to output the button or just the target element.
+	 * @param string $aria_label      The aria-label attribute for the button.
+	 *
+	 * @return string
+	 */
+	function tribe_copy_to_clipboard_button( string $content_to_copy, bool $output_button = true, string $aria_label = '' ): string {
+		static $counter = 1;
```

Comment:
We should switch this to storing the counter in our container via Tribe__Cache:

```php
$cache_key = 'tec_copy_to_clipboard_counter';
$counter = tribe( 'cache' )->get( $cache_key, '', 1 );

// Do stuff.

tribe( 'cache' )->set( $cache_key, ++$counter, Tribe__Cache::NON_PERSISTENT );
```

This should help with automated testing value resetting.